### DISCUSSION TITLE
How to default Vim :Files command to open file in a new tab?

### DISCUSSION THREAD

**Discussion by tonyvitonis** [junegunn/fzf#2519] [Q&A] [answered]
When I use Vim, I prefer to use tabs instead of buffers. For that reason, I'd like the default action of :Files to be "open the file in a new tab". I've dug through the code and doc and tried a few things on a private branch, but I can't seem to make it work. Is there any way to do it?

**Comment by j-xella** [comment]
Hi,

Just to be clear, in vim "buffers" refer to actual documents that vim currently works with. A document is often a file, but could also be some automatically generated text that exists in VIM memory only. Same buffer can be open in multiple tabs, for example. You are probably talking about "windows" in VIM terminology - a rectangle inside vim that shows some buffer.

I could not find tab support in the source code of fzf.vim either. Moreover, VIM's "tab" command does not work with :Files too. That being said, you could cheat the system with:

:tabnew | Files

**Reply by tonyvitonis** [reply]
Thanks for the reply. I'm clear on the difference between buffers and windows and tabs. My question indirectly refers to this in the documentation:

"Most commands support **CTRL-T** / CTRL-X / CTRL-V key bindings to open in a **new tab**, a new split, or in a new vertical split".

When you enter :Files, make a selection and hit Ctrl-T, the file is shown in a new _tab_ in the Vim session. I'd like to make that the default behavior when I hit Enter (instead of Ctrl-T).

That said, ":tabnew | Files" isn't a bad stopgap measure. I'll assign it to a key and see how it feels.

**Comment by junegunn** [comment, marked as answer]
You can set up `g:fzf_action` as described in https://github.com/junegunn/fzf/blob/master/README-VIM.md#configuration

```vim
let g:fzf_action = { 'enter': 'tab split' }
```

**Reply by tonyvitonis** [reply]
This is exactly what I was looking for. Thank you very much.

**Reply by dtrckd** [reply]
What is the right shortcut to open in new tab with <c-enter>. 
Is it possible ?
